tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -395,9 +399,343 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 		// TODO: Verify no orphan volumes are created
 	})
 
+	/*
+		Verify EZT online volume expansion to a large size (should take >vpxd task timeout)
+		Steps:
+		1	Create a SPBM policy with EZT volume allocation
+		2	Create a SC using policy created in step 1 and allowVolumeExpansion set to true
+		3	Create a 2g PVC using SC created in step 2, say pvc1
+		4	Wait and verify for pvc1 to be bound
+		5	Create a pod using pvc1 say pod1 and wait for it to be ready
+		6	Expand pvc1 to a large size this should take more than vpxd timeout
+		7	Wait and verify the file system resize on pvc1
+		8	Delete pod1
+		9	Delete pvc1
+		10	Delete the SC created in step 2
+		11	Deleted the SPBM policy created in step 1
+	*/
+	t := "[csi-block-vanilla] Verify EZT online volume expansion to a large size which takes longer than vpxd timeout"
+	ginkgo.It(t, func() {
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
+		if sharedvmfsURL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		}
+
+		largeSize := os.Getenv(envDiskSizeLarge)
+		if largeSize == "" {
+			largeSize = diskSizeLarge
+		}
+
```

Comment:
It's better to either use a default value or fail/skip this test if env variable is not present, as we are setting a particular vpxd timeout according to the pvc size